src/components/Button/Button.js

```diff
@@ -1,16 +1,6 @@
-import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
 
-Button.propTypes = {
-  children: PropTypes.node,
-  onClick: PropTypes.func.isRequired,
-}
-
-export default function Button(props) {
-  return <ButtonStyled {...props} />
-}
-
-const ButtonStyled = styled.button`
```

Comment:
If you delete this, then you won't be able to call the action in storybook. I mean it's not mandatory to do that, but maybe it would be nice to see if the button is clickable in storybook, no?